pallets/connectors-gateway/connectors-gateway-routers/Cargo.toml

```diff
@@ -0,0 +1,100 @@
+[package]
+authors = ["Centrifuge <admin@centrifuge.io>"]
+description = "Centrifuge Connectors Gateway Routers"
+edition = "2021"
+license = "LGPL-3.0"
+name = "connectors-gateway-routers"
```

Comment:
We don't need to do this now but the current directory structure for all things connectors is getting a bit out of hand. For this PR we could just rename this crate directory to become `pallets/connectors-gateway/routers/`and keep the crate name itself as is, `connectors-gateway-routers`.

Reply:
Agree, gonna move some of these around until I get a better structure.

Reply:
Changed, please let me know what you think.